### DISCUSSION TITLE
SongPi - Continuous song recognition app using ShazamIO (Python)

### DISCUSSION THREAD

**Discussion by Mildywot** [shazamio/ShazamIO#125] [Show and tell]
I made a continuous song recognition GUI app built with Python which uses ShazamIO's code, GitHub here:
[SongPi](https://github.com/Mildywot/SongPi/)

Sample images and how the code works is available on GitHub.

It runs on Raspberry Pi and Windows, enjoy!

**Comment by lonix1** [comment]
Windows only?

**Reply by Mildywot** [reply]
I've tested it on a few Raspberry Pi's (Linux) and on Windows, but should technically run on anything that supports Python.
